### PR TITLE
Enki bot js

### PR DESCRIPTION
**Pull request by elliehoward**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/intro-js/js-practice-console-in.md?ref=67fb9c88a0370e5f6870e92ae3633c163eba0dd1
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/intro-js/js-practice-using-javascript-on-a-webpage-inline.md?ref=67fb9c88a0370e5f6870e92ae3633c163eba0dd1
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/intro-js/js-practice-using-javascript-on-a-webpage.md?ref=67fb9c88a0370e5f6870e92ae3633c163eba0dd1
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/syntax/double-and-single-quotes.md?ref=67fb9c88a0370e5f6870e92ae3633c163eba0dd1
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/syntax/javascript-basics-part-2.md?ref=67fb9c88a0370e5f6870e92ae3633c163eba0dd1
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/types/be-careful-when-using-typeof.md?ref=67fb9c88a0370e5f6870e92ae3633c163eba0dd1
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/types/js-practice-booleans.md?ref=67fb9c88a0370e5f6870e92ae3633c163eba0dd1
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/types/js-practice-hoisting.md?ref=67fb9c88a0370e5f6870e92ae3633c163eba0dd1
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/types/js-practice-taking-precedence.md?ref=67fb9c88a0370e5f6870e92ae3633c163eba0dd1
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/types/taking-precedence.md?ref=67fb9c88a0370e5f6870e92ae3633c163eba0dd1
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/types/types-hoisting.md?ref=67fb9c88a0370e5f6870e92ae3633c163eba0dd1
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/types/types-introduction.md?ref=67fb9c88a0370e5f6870e92ae3633c163eba0dd1
- [ ] https://enkipro.com/preview-insight/enkidevs/curriculum/javascript/core/types/types-string.md?ref=67fb9c88a0370e5f6870e92ae3633c163eba0dd1

File: javascript/core/types/js-practice-booleans.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ tags:
 
 type: exercise
 
-link: http://www.codewars.com/kata/571edd157e8954bab500032d
-linkType: codewars
+link: https://jsfiddle.net/elliehoward/m9gwybxa/
```
Comment (lizTheDeveloper) [inline javascript/core/types/js-practice-booleans.md]: Will you use the glitch exercise that goes with this

File: javascript/core/intro-js/js-practice-console-in.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ type: exercise
 
 linkType: jsfiddle
 
-link: https://jsfiddle.net/elliehoward/uw19xmkv/5/
+link: https://jsfiddle.net/elliehoward/uw19xmkv/
```
Comment (lizTheDeveloper) [inline javascript/core/intro-js/js-practice-console-in.md]: Use the nodeschool glitch one for here as well

**Comment by lizTheDeveloper** [comment]
merged

**Pull request closed without merging**